rust/src/dns/dns.rs

```diff
@@ -437,6 +437,7 @@ impl DNSState {
             input,
             input.len() as i64,
             DnsFrameType::Pdu as u8,
+            None,
```

Comment:
@jasonish do you want to set tx_id for dns frames ?

Reply:
Yes, and it probably makes sense for all right?  Especially if we add more frames and recommend people using them.

Reply:
It makes sense for DNS and websocket where one suricata transaction is one PDU.

I am not so sure for RFB, SMB and others... I am not sure that every byte on the wire belongs to a transaction for instance...